docker-compose

```diff
@@ -0,0 +1,33 @@
+yarn add v1.22.5
```

Comment:
This looks like some kind of output file.  I'm guessing we don't need this in version control?

Reply:
When attempting to delete this Pycharm is throwing up warnings since I have line 21 in .travil.yml

Reply:
Disregard it let me do the refactor